src/render.js

```diff
@@ -11,9 +13,27 @@ import cheerio from 'cheerio';
  * thus I'd like to keep this API in here even though it's not really "ours".
  *
  * @param node
+ * @param options
  * @returns {Cheerio}
  */
-export default function render(node) {
+export default function render(node, options = {}) {
+  if (options.context && (node.type.contextTypes || options.childContextTypes)) {
+    const childContextTypes = node.type.contextTypes || {};
+    if (options.childContextTypes) {
+      objectAssign(childContextTypes, options.childContextTypes);
+    }
+    class ContextWrapper extends React.Component {
+      getChildContext() {
+        return options.context;
+      }
+      render() {
+        return node;
+      }
+    }
```

Comment:
Any way we can avoid defining the class every time `render` is invoked? I guess its not a huge deal if it's required. Maybe we can move this logic out of the `render` method into something like `renderWithContext` to make the actual `render` call a little cleaner

```js
if (options.childContextTypes) {
  return renderWithContext(node, options)
}
```

What do you think?